src/OpenSpaceToolkit/Astrodynamics/Access/Generator.cpp

```diff
@@ -538,6 +538,60 @@ Generator                       Generator::AerRanges                        (
 
 }
 
+Generator                       Generator::AerMask                           (  const   Map<Real, Real>&            anAzimuthElevationMask,
+                                                                                const   Interval<Real>&             aRangeRange,
+                                                                                const   Environment&                anEnvironment                               )
+{                    
+
+    using ostk::core::types::Real ;
+    using ostk::core::ctnr::Map ;
+
+    if ((anAzimuthElevationMask.empty()) || (anAzimuthElevationMask.begin()->first < 0.0) || (anAzimuthElevationMask.rbegin()->first > 360.0))
+    {
+        throw ostk::core::error::runtime::Wrong("Azimuth-Elevation Mask") ;
+    }
+
+    for (const auto& azimuthElevationPair : anAzimuthElevationMask) {
+        if ((azimuthElevationPair.second).abs() > 90.0) 
+        {
+            throw ostk::core::error::runtime::Wrong("Azimuth-Elevation Mask") ;
+        }
+    }
+
+    Map<Real, Real> anAzimuthElevationMask_deg = anAzimuthElevationMask ;
+    const Interval<Real> rangeRange_m = aRangeRange ;
+
+    if (anAzimuthElevationMask_deg.begin()->first != 0.0)
+    {
+        anAzimuthElevationMask_deg.insert({ 0.0, anAzimuthElevationMask_deg.begin()->second }) ;
+    }
+
+    if (anAzimuthElevationMask_deg.rbegin()->first != 360.0)
+    {
+        anAzimuthElevationMask_deg.insert({ 360.0, anAzimuthElevationMask_deg.begin()->second }) ;
```

Comment:
`anAzimuthElevationMask_deg.rbegin()->second }) ;` maybe?

Reply:
This ensures continuity of elevation at azimuth `0.0` and `360.0`.

Needed because tested azimuth values larger than the largest azimuth value provided by the mask (i.e. `anAzimuthElevationMask_deg.rbegin()->first`) need an upper bound for the calculation below.